test/SILOptimizer/diagnostic_constant_propagation_int_arch64.swift

```diff
@@ -0,0 +1,235 @@
+// RUN: %target-swift-frontend -emit-sil -primary-file %s -o /dev/null -verify
+//
+// REQUIRES: CPU=x86_64 || CPU=arm64 || CPU=powerpc64 || CPU=powerpc64le || CPU=s390x
```

Comment:
I wonder it might be better to use PTRSIZE=64 and PTRSIZE=32. Pointer size and Int size aren't the same thing -- but this would avoid having to enumerate all 64-bit CPUs